packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md

```diff
@@ -1,48 +1,43 @@
-    * `nonce`: Use any value. This is a string that is included in the returned ID token. It associates a client session with an ID token and mitigates replay attacks. In this example, we aren't requesting the `id_token` response type, so an ID token isn't returned.
-
-3. Enter your credentials on the Okta Sign-in Page that appears. After you authenticate, the browser returns an access token in the address bar.
+1. In Postman, select the request that you want to make, such as a `GET` request to the `/api/v1/users` endpoint to get back a list of all users.
+2. On the **Header** tab, remove the existing SSWS Authorization API Key.
+3. Click the **Authorization** tab and from the **Type** drop-down list, select **OAuth 2.0**.
+4. On the right, click **Get New Access Token**.
+5. In the dialog box that appears, enter a name for the token and select **Implicit** as the grant type.
+6. Define the following for the token request:
 
-    In this example, you previously added the `okta.users.read` scope to the application's grants collection. You then requested that scope be included in the access token. Since this is an admin-level scope, you need to sign in with admin user credentials that are associated with your app.
-
-    The response that is returned looks something like this (the token is truncated for brevity):
-
-    ```bash
-    http://{yourOktadomain}/authorization-code/callback#access_token=eyJraWQiOiJEa1lUbmhTdkd5OEJk.......Y5D6XG&token_type=Bearer&expires_in=3600&scope=okta.users.read&state=1234
-    ```
+    * **Callback URL**: Define the callback location where Okta returns the token after the user finishes authenticating. This URL must match one of the redirect URIs that you configured in the <GuideLink link="../create-an-OAuth-2.0-app-in-okta">Create an OAuth 2.0 app in Okta</GuideLink> section.
+    * **Auth URL**: Enter the authorization endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/authorize?none=WBYA`
+        > **Note:** There isn't an entry in the dialog box for a `nonce` parameter. But, you can append it to the Auth URL here. Use any value for `nonce`. This is a string that is included in the returned ID token. It associates a client session with an ID token and mitigates replay attacks. In this example, we aren't requesting the `id_token` response type, so an ID token isn't returned.
+    * **Client ID**: Use the `client_id` of your Okta OAuth application that you created in the <GuideLink link="../create-oauth-app">previous step</GuideLink>.
+    * **Scope**: Include the scopes that allow you to perform the actions on the endpoint that you want to access. The scopes requested for the access token must already be in the application's grants collection and the user must have the permission to perform those actions. See <GuideLink link="../scopes">Scopes & supported endpoints</GuideLink> for more information.
```

Comment:
"must already **exist** in the application's **grant** collection" (nit picking on the first, not sure if "grants collection" is the noun for the second)

Reply:
thanks on the "be" removal and it is grants collection.